### ISSUE TITLE
[Bug]: HKSV Doesn't Start with 1080p Battery Doorbell

### ISSUE THREAD

**Issue by louisefindlay23** [homebridge-eufy-security/plugin#184]
### What happened?

Connected my Eufy 1080p battery doorbell using a new admin account and a brand new Homebridge instance via unbridged mode. Tried to open the HKSV stream in the Home app but it errored out (see the logs). 

RTSP is set to false automatically since the doorbell doesn't support it but the log says `The write handler for the characteristic 'Selected RTP Stream Configuration' on the accessory 'Doorbell 5A70' didn't respond at all!`. Quality is set to auto in the Eufy app and I tried to setting Advanced Video Config to Performance but nothing changed.

Seems to be similar to #152 but not the same. Provided logs below with plugin debug enabled.

### Device Type

Doorbell (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v16 (Supported)

### Operating System

Docker (UnSupported)

### Relevant log output

```shell
[10/16/2022, 4:07:39 PM] [EufySecurity-2.1.0] DEBUG: Doorbell RTSP rtspStream:false
[10/16/2022, 4:07:39 PM] [EufySecurity-2.1.0] DEBUG: Doorbell Looks like not compatible with RTSP
[10/16/2022, 4:07:39 PM] [EufySecurity-2.1.0] DEBUG: Doorbell New instance requests livestream. There were 0 instance(s) using the livestream until now.
[10/16/2022, 4:07:39 PM] [EufySecurity-2.1.0] DEBUG: Doorbell Start new station livestream (P2P Session)...
[10/16/2022, 4:07:48 PM] [homebridge-eufy-security] This plugin slows down Homebridge. The write handler for the characteristic 'Selected RTP Stream Configuration' on the accessory 'Doorbell 5A70' didn't respond at all!. Please check that you properly call the callback! See https://homebridge.io/w/JtMGR for more info.
[10/16/2022, 4:07:48 PM] [EufySecurity-2.1.0] DEBUG: Doorbell Receive Apple HK Stop request{"sessionID":"17265525-bfe6-4e3e-8696-bbd18ed5061f","type":"stop"}
[10/16/2022, 4:07:48 PM] [EufySecurity-2.1.0] DEBUG: Stopping session with id: 17265525-bfe6-4e3e-8696-bbd18ed5061f
[10/16/2022, 4:07:48 PM] [EufySecurity-2.1.0] DEBUG: No session to stop.
[10/16/2022, 4:07:49 PM] [EufySecurity-2.1.0] ERROR: Doorbell Local livestream didn't start in time. Abort livestream request.
[10/16/2022, 4:07:49 PM] [EufySecurity-2.1.0] ERROR: Doorbell Unable to start the livestream: no started livestream found
[10/16/2022, 4:07:50 PM] [EufySecurity-2.1.0] DEBUG: Doorbell Device appears to be inactive. Stopping video stream.
[10/16/2022, 4:07:50 PM] [EufySecurity-2.1.0] DEBUG: Stopping session with id: 17265525-bfe6-4e3e-8696-bbd18ed5061f
[10/16/2022, 4:07:50 PM] [EufySecurity-2.1.0] DEBUG: No session to stop.
```

### config

```shell
{
    "bridge": {
        "name": "Homebridge 9B8B",
        "username": "XYZ",
        "port": 0000,
        "pin": "XYZ",
        "advertiser": "bonjour-hap"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 0000,
            "auth": "form",
            "theme": "orange",
            "tempUnits": "c",
            "lang": "auto",
            "platform": "config"
        },
        {
            "username": "XYZ",
            "password": "XYZ",
            "country": "GB",
            "cameras": [
                {
                    "serialNumber": "0000",
                    "rtsp": false,
                    "enableCamera": true,
                    "videoConfig": {
                        "audio": true,
                        "maxWidth": 640,
                        "maxHeight": 480,
                        "maxFPS": 15,
                        "useSeparateProcesses": true
                    },
                    "enableButton": false,
                    "motionButton": false,
                    "unbridge": true,
                    "snapshotHandlingMethod": 3
                }
            ],
            "platform": "EufySecurity",
            "ignoreStations": [],
            "ignoreDevices": [],
            "cleanCache": true,
            "enableDetailedLogging": true
        }
    ]
}
```

**Comment by edm0d0n2** [comment]
Did you managed to fix this?

**Comment by louisefindlay23** [comment]
> Did you managed to fix this?

Unfortunately not, @edm0d0n2.

**Comment by Dravegon** [comment]
@louisefindlay23

I got HKSV working on the Battery Doorbell without RTSP obviously.

For starters idk if your advertiser: bonjour-hap and Homebridge 1.4.x will work, I updated to advertiser: Avahi and Homebridge 1.5.0.

Also since the doorbell doesn’t support RTSP don’t mess with the Experimental features.

My settings:(from the top of my head)
Eufy app;
Video quality:High
Codec:Low(h.264)

Plugin settings;
enable: prefer local connection(it’s at the plugin settings not the doorbell settings)

Advanced Video Config;
i have it set to Copy(since i’m only running a Rpi 4B)
I enabled the audio codec setting too just on default settings(not sure if that did the trick to get HKSV running)

Oh and if i recall correctly use the latest beta 5 of the plugin for HKSV.
